dashboard/src/app/index.module.ts

```diff
@@ -49,11 +49,19 @@ window.name = 'NG_DEFER_BOOTSTRAP!';
 
 declare const Keycloak: Function;
 function buildKeycloakConfig(keycloakSettings: any) {
-  return {
-    url: keycloakSettings['che.keycloak.auth_server_url'],
-    realm: keycloakSettings['che.keycloak.realm'],
-    clientId: keycloakSettings['che.keycloak.client_id']
-  };
+  const theOidcProvider = keycloakSettings['che.keycloak.oidc_provider'];
+  if(!theOidcProvider) {
```

Comment:
I think there is a missing space between if and parenthesis
Also could you confirm that there is no lint report from dashboard build around the UD changes ?

Reply:
fixed in commit https://github.com/eclipse/che/pull/8614/commits/b4604216a28c11a500d754e81df000213827762f